src/lib/subname-helpers.ts

```diff
@@ -1,16 +1,17 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
 // TODO: pull from ens utils lib or something
 export const ROOT_NODE = namehash("");
+export const ETH_NODE = namehash("eth");
 
-// TODO: this should probably be a part of some ens util lib
+// TODO: this should probably be a part of some ens util lib but isn't exposed by viem or ensjs
 export const makeSubnodeNamehash = (node: Hex, label: Hex) => keccak256(concat([node, label]));
 
-// https://github.com/wevm/viem/blob/main/src/utils/ens/encodeLabelhash.ts
-export const encodeLabelhash = (hash: Hex): `[${string}]` => `[${hash.slice(2)}]`;
-
+// converts uint256-encoded nodes to hex
 export const tokenIdToLabel = (tokenId: bigint) => toHex(tokenId, { size: 32 });
 
+// TODO: this should be part of some more standard ens helper lib
 // https://github.com/ensdomains/ens-subgraph/blob/master/src/utils.ts#L68
 const INVALID_CHARACTER_CODES_IN_LABEL = new Set([0, 46, 91, 93]);
 export const isLabelValid = (name: string) => {
```

Comment:
```suggestion
export const isLabelUnindexable = (name: string) => {
```

@shrugs Also a bit confused by the param `name`. Should we update that param to be `label`?

Reply:
subgraph util called it name, so that's a vestige, but agreed it should be label